### ISSUE TITLE
Restart button just reloads "Cloud Login" window on 12.6 -> 13.0.1 Upgrade

### ISSUE THREAD

**Issue by PeetMcK** [twocanoes/xcreds#60]
I'm experiencing an odd behavior on a freshly upgraded to Ventura M1 Mac mini. IT looks very much like the auth.db has been reset during the OS upgrade, but I don't have a copy of the pre Ventura settings ... here's where it is now. 
 
```
# /usr/bin/security authorizationdb read system.login.console
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
    <key>class</key>
    <string>evaluate-mechanisms</string>
    <key>comment</key>
    <string>Login mechanism based rule.  Not for general use, yet.</string>
    <key>created</key>
    <real>635213377.92312098</real>
    <key>mechanisms</key>
    <array>
        <string>builtin:prelogin</string>
        <string>builtin:policy-banner</string>
        <string>XCredsLoginPlugin:LoginWindow</string>
        <string>builtin:login-begin</string>
        <string>builtin:reset-password,privileged</string>
        <string>loginwindow:FDESupport,privileged</string>
        <string>builtin:forward-login,privileged</string>
        <string>builtin:auto-login,privileged</string>
        <string>builtin:authenticate,privileged</string>
        <string>PKINITMechanism:auth,privileged</string>
        <string>builtin:login-success</string>
        <string>loginwindow:success</string>
        <string>HomeDirMechanism:login,privileged</string>
        <string>HomeDirMechanism:status</string>
        <string>MCXMechanism:login</string>
        <string>CryptoTokenKit:login</string>
        <string>PSSOAuthPlugin:login-auth</string>
        <string>loginwindow:done</string>
    </array>
    <key>modified</key>
    <real>690923298.77724099</real>
    <key>shared</key>
    <true/>
    <key>tries</key>
    <integer>10000</integer>
    <key>version</key>
    <integer>10</integer>
</dict>
</plist>
YES (0)
```
vs XCreds installed directly on Ventura: 

```
 % /usr/bin/security authorizationdb read system.login.console
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>class</key>
	<string>evaluate-mechanisms</string>
	<key>comment</key>
	<string>Login mechanism based rule.  Not for general use, yet.</string>
	<key>created</key>
	<real>688437184.77241194</real>
	<key>mechanisms</key>
	<array>
		<string>builtin:prelogin</string>
		<string>builtin:policy-banner</string>
		<string>XCredsLoginPlugin:LoginWindow</string>
		<string>XCredsLoginPlugin:PowerControl,privileged</string>
		<string>builtin:login-begin</string>
		<string>XCredsLoginPlugin:CreateUser,privileged</string>
		<string>builtin:reset-password,privileged</string>
		<string>loginwindow:FDESupport,privileged</string>
		<string>builtin:forward-login,privileged</string>
		<string>builtin:auto-login,privileged</string>
		<string>builtin:authenticate,privileged</string>
		<string>PKINITMechanism:auth,privileged</string>
		<string>builtin:login-success</string>
		<string>loginwindow:success</string>
		<string>HomeDirMechanism:login,privileged</string>
		<string>HomeDirMechanism:status</string>
		<string>MCXMechanism:login</string>
		<string>CryptoTokenKit:login</string>
		<string>loginwindow:done</string>
		<string>XCredsLoginPlugin:LoginDone</string>
		<string>XCredsLoginPlugin:EnableFDE,privileged</string>
		<string>XCredsLoginPlugin:KeychainAdd,privileged</string>
	</array>
	<key>modified</key>
	<real>689720121.40980196</real>
	<key>shared</key>
	<true/>
	<key>tries</key>
	<integer>10000</integer>
	<key>version</key>
	<integer>9</integer>
</dict>
</plist>
YES (0)
```
I have many Ventura upgrades to go, so I'll test again. Just noting the issue.

**Comment by PeetMcK** [comment]
After reinstalling XCreds 2.0 on the affected mini ...

```
# /usr/bin/security authorizationdb read system.login.console
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
    <key>class</key>
    <string>evaluate-mechanisms</string>
    <key>comment</key>
    <string>Login mechanism based rule.  Not for general use, yet.</string>
    <key>created</key>
    <real>635213377.92312098</real>
    <key>mechanisms</key>
    <array>
        <string>builtin:prelogin</string>
        <string>builtin:policy-banner</string>
        <string>XCredsLoginPlugin:LoginWindow</string>
        <string>XCredsLoginPlugin:PowerControl,privileged</string>
        <string>builtin:login-begin</string>
        <string>XCredsLoginPlugin:CreateUser,privileged</string>
        <string>builtin:reset-password,privileged</string>
        <string>loginwindow:FDESupport,privileged</string>
        <string>builtin:forward-login,privileged</string>
        <string>builtin:auto-login,privileged</string>
        <string>builtin:authenticate,privileged</string>
        <string>PKINITMechanism:auth,privileged</string>
        <string>builtin:login-success</string>
        <string>loginwindow:success</string>
        <string>HomeDirMechanism:login,privileged</string>
        <string>HomeDirMechanism:status</string>
        <string>MCXMechanism:login</string>
        <string>CryptoTokenKit:login</string>
        <string>PSSOAuthPlugin:login-auth</string>
        <string>loginwindow:done</string>
        <string>XCredsLoginPlugin:LoginDone</string>
        <string>XCredsLoginPlugin:EnableFDE,privileged</string>
        <string>XCredsLoginPlugin:KeychainAdd,privileged</string>
    </array>
    <key>modified</key>
    <real>690924704.37308395</real>
    <key>shared</key>
    <true/>
    <key>tries</key>
    <integer>10000</integer>
    <key>version</key>
    <integer>10</integer>
</dict>
</plist>
YES (0)
```

**Comment by twocanoes** [comment]
There is a fix for this in 3.1
